x-pack/plugins/maps/public/routing/routes/list/maps_list_view.js

```diff
@@ -388,14 +396,12 @@ export class MapsListView extends React.Component {
     let createButton;
     if (!this.state.readOnly) {
       createButton = (
-        <Link to={'/map'}>
-          <EuiButton data-test-subj="newMapLink" fill>
-            <FormattedMessage
-              id="xpack.maps.mapListing.createMapButtonLabel"
-              defaultMessage="Create map"
-            />
-          </EuiButton>
-        </Link>
+        <EuiButton data-test-subj="newMapLink" onClick={navigateToNewMap}>
```

Comment:
navigation is much better but `fill` was removed.

<img width="400" alt="Screen Shot 2020-09-01 at 2 39 14 PM" src="https://user-images.githubusercontent.com/373691/91903723-f0d92680-ec60-11ea-9466-b7eeb834f12c.png">